service/interfaces.go

```diff
@@ -2,26 +2,30 @@ package service
 
 import (
 	"context"
-	"github.com/ONSdigital/dp-image-importer/api"
+	"github.com/ONSdigital/dp-image-importer/event"
 	"net/http"
 
 	"github.com/ONSdigital/dp-healthcheck/healthcheck"
 	"github.com/ONSdigital/dp-image-importer/config"
+	kafka "github.com/ONSdigital/dp-kafka"
+	"github.com/aws/aws-sdk-go/aws/session"
 )
 
 //go:generate moq -out mock/initialiser.go -pkg mock . Initialiser
 //go:generate moq -out mock/server.go -pkg mock . HTTPServer
 //go:generate moq -out mock/healthCheck.go -pkg mock . HealthChecker
 
```

Comment:
maybe group go generate commands together by removing whitespace between line 16 and 18?